README.md

```diff
@@ -37,7 +37,6 @@ A curated list of awesome tools, platforms, products, and resources for Managed
 - [ConnectWise Automate](https://www.connectwise.com/software/automate) - RMM software that automates the technical work & improves ROI.
 - [Datto RMM](https://www.datto.com/rmm) - Managed service providers can remotely secure, monitor, and manage endpoints.
 - [Kaseya VSA](https://www.kaseya.com/products/vsa) - Unified endpoint management, from traditional to emerging – including mobile IOT, VMs and more.
-- [Level](https://level.io) - Flexible, intuitive, and secure remote monitoring and management (RMM) platform.
```

Comment:
I believe this is a mistake. Please rebase your branch, or recreate it from scratch.